### ISSUE TITLE
Magento 2.4.4 support 

### ISSUE THREAD

**Issue by mngoe** [owebia/magento2-module-shared-php-config#8]
Hello , 

I have a problem with your module.
I use "owebia/magento2-module-advanced-shipping-pro": "^1.1.13" (so basically the 3.0.7 version of magento2-module-shared-php-config).

On a Magento 2.4.4 with PHP 8.1, I have this error : 

```
[2023-04-22T08:52:56.287720+00:00] main.CRITICAL: Exception: Deprecated Functionality: hash(): Passing null to parameter #2
 ($data) of type string is deprecated in /var/www/html/vendor/owebia/magento2-module-shared-php-config/Helper/Config.php on line 82 in /var/www/html/vendor/magento/framework/App/ErrorHandler.php:61
Stack trace:
#0 [internal function]: Magento\Framework\App\ErrorHandler->handler()
#1 /var/www/html/vendor/owebia/magento2-module-shared-php-config/Helper/Config.php(82): hash()
#2 /var/www/html/vendor/owebia/magento2-module-adv-ship-universal-setting-addon/Model/Plugin/ShippingPlugin.php(85): Owebia\SharedPhpConfig\Helper\Config->parse()
#3 /var/www/html/vendor/magento/framework/Interception/Interceptor.php(135): Owebia\AdvShipUniversalSettingAddon\Model\Plugin\ShippingPlugin->aroundCollectRates()
#4 /var/www/html/vendor/magento/framework/Interception/Interceptor.php(153): Magento\Shipping\Model\Shipping\Interceptor->Magento\Framework\Interception\{closure}()
#5 /var/www/html/generated/code/Magento/Shipping/Model/Shipping/Interceptor.php(23): Magento\Shipping\Model\Shipping\Interceptor->___callPlugins()
#6 /var/www/html/vendor/magento/module-quote/Model/Quote/Address.php(1077): Magento\Shipping\Model\Shipping\Interceptor->collectRates()
#7 /var/www/html/vendor/magento/module-quote/Model/Quote/Address.php(1004): Magento\Quote\Model\Quote\Address->requestShippingRates()
#8 /var/www/html/vendor/magento/module-quote/Model/Quote/Address/Total/Shipping.php(83): Magento\Quote\Model\Quote\Address->collectShippingRates()
#9 /var/www/html/vendor/magento/module-quote/Model/Quote/TotalsCollector.php(261): Magento\Quote\Model\Quote\Address\Total\Shipping->collect()
#10 /var/www/html/vendor/magento/module-quote/Model/Quote/TotalsCollector.php(156): Magento\Quote\Model\Quote\TotalsCollector->collectAddressTotals()
#11 /var/www/html/vendor/magento/framework/Interception/Interceptor.php(58): Magento\Quote\Model\Quote\TotalsCollector->collect()
#12 /var/www/html/vendor/magento/framework/Interception/Interceptor.php(138): Magento\Quote\Model\Quote\TotalsCollector\Interceptor->___callParent()
#13 /var/www/html/vendor/magento/framework/Interception/Interceptor.php(153): Magento\Quote\Model\Quote\TotalsCollector\Interceptor->Magento\Framework\Interception\{closure}()
#14 /var/www/html/generated/code/Magento/Quote/Model/Quote/TotalsCollector/Interceptor.php(23): Magento\Quote\Model\Quote\TotalsCollector\Interceptor->___callPlugins()
#15 /var/www/html/vendor/magento/module-quote/Model/Quote.php(2015): Magento\Quote\Model\Quote\TotalsCollector\Interceptor->collect()
#16 /var/www/html/vendor/magento/module-quote/Model/QuoteRepository/SaveHandler.php(121): Magento\Quote\Model\Quote->collectTotals()
#17 /var/www/html/generated/code/Magento/Quote/Model/QuoteRepository/SaveHandler/Interceptor.php(23): Magento\Quote\Model\QuoteRepository\SaveHandler->save()
....
```

So basically on /Helper/Config.php on line 82 this line is 
`$hash = hash('md5', $configuration);`
can have a $configuration null value, and so 8.1 now raise a Critical error blocking cart process.

I add a check on this $configuration value manually on the vendor , but obviously is not the correct method to solve.

**Comment by owebia** [comment]
Hi,

Thanks for your feedback.
We will provide a fix in the next release.

Best Regards,
A.L.

**Comment by owebia** [comment]
Hi

This issue has been fixed in the 3.0.8 release of this module.

Best Regards,
A.L.
